exporters/otlp/profiles/src/main/java/io/opentelemetry/exporter/otlp/profiles/ProfilesExporter.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.otlp.profiles;
+
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.SpanProcessor;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import java.io.Closeable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+// from sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/ProfilesExporter.java
+// despite what the comments say this isn't yet wired to ths sdk!
+
+/**
+ * An interface that allows different profiling services to export recorded data in their own
+ * format.
+ *
+ * <p>To export data this MUST be register to the {@code TracerSdk} using a {@link
+ * SimpleSpanProcessor} or a {@code BatchSampledSpansProcessor}.
+ */
+public interface ProfilesExporter extends Closeable {
```

Comment:
The other exporters are singular 
```suggestion
public interface ProfileExporter extends Closeable {
```